craco.config.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable prefer-import/prefer-import-over-require */
+const path = require('path');
+const BabelRcPlugin = require('@jackwilsdon/craco-use-babelrc');
+
+module.exports = {
+    style: {
+        postcss: {
+            mode: 'file',
+        },
+    },
+    plugins: [
+        { plugin: BabelRcPlugin },
+        {
+            plugin: {
+                overrideWebpackConfig: ({ webpackConfig, context: { env } }) => {
+                    const options = {
+                        test: /\.css$/,
+                        include: [
```

Comment:
Would it be possible to find a more generic name for the `include` attribute, or the `options` object? It doesn't seem readable to me to assign the `include` to the `moduleCssRule.include` and the `cssRule.exclude`.

Maybe call the object `newModuleCssOptions`?

Reply:
I named it as `options` since I was thinking to start a plugin and one can pass options to the plugin. But I'll rename to `newModuleCssRule` as it makes more sense for now. What do you think?